<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetTickOptionsDialog</class>
 <widget class="QDialog" name="SetTickOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tick options</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Major X ticks (Geocode)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QCheckBox" name="drawMajorXTicklines_checkBox">
        <property name="text">
         <string>Ticklines</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="drawMajorXTicksLeft_checkBox">
        <property name="text">
         <string>Ticks left</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="drawMajorXTicksRight_checkBox">
        <property name="text">
         <string>Ticks right</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="drawMajorXTicksInside_checkBox">
        <property name="text">
         <string>Ticks inside</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="drawMajorXTicksOutside_checkBox">
        <property name="text">
         <string>Ticks outside</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Major Y ticks (Parameter)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QCheckBox" name="drawMajorYTicklines_checkBox">
        <property name="text">
         <string>Ticklines</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="drawMajorYTicksTop_checkBox">
        <property name="text">
         <string>Ticks top</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="drawMajorYTicksBottom_checkBox">
        <property name="text">
         <string>Ticks bottom</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="drawMajorYTicksInside_checkBox">
        <property name="text">
         <string>Ticks inside</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="drawMajorYTicksOutside_checkBox">
        <property name="text">
         <string>Ticks outside</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Minor X ticks (Geocode)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="drawMinorXTicklines_checkBox">
        <property name="text">
         <string>Ticklines</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="drawMinorXTicksLeft_checkBox">
        <property name="text">
         <string>Ticks left</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="drawMinorXTicksRight_checkBox">
        <property name="text">
         <string>Ticks right</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="drawMinorXTicksInside_checkBox">
        <property name="text">
         <string>Ticks inside</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="drawMinorXTicksOutside_checkBox">
        <property name="text">
         <string>Ticks outside</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Number of minor ticks
between major ticks</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="numMinorXTicks_spinBox">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Minor Y ticks (Parameter)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="drawMinorYTicklines_checkBox">
        <property name="text">
         <string>Ticklines</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="drawMinorYTicksTop_checkBox">
        <property name="text">
         <string>Ticks top</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="drawMinorYTicksBottom_checkBox">
        <property name="text">
         <string>Ticks bottom</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="drawMinorYTicksInside_checkBox">
        <property name="text">
         <string>Ticks inside</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="drawMinorYTicksOutside_checkBox">
        <property name="text">
         <string>Ticks outside</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Number of minor ticks
between major ticks</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="numMinorYTicks_spinBox">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Tick options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Tickmark length</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="TickmarkLength_spinBox">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Tickmark width</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="TickmarkWidth_spinBox">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Tickline width</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="TicklineWidth_spinBox">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Minor tickmark width</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="MinorTickmarkWidth_spinBox">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Minor tickline width</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="MinorTicklineWidth_spinBox">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OK_pushButton">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel_pushButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="TickColors_groupBox">
     <property name="title">
      <string>Tick colors</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QPushButton" name="TickmarkColor_pushButton">
        <property name="text">
         <string>Tickmark color</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="TickmarkColor_frame">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="TicklineColor_pushButton">
        <property name="text">
         <string>Tickline color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFrame" name="TicklineColor_frame">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="MinorTicklineColor_pushButton">
        <property name="text">
         <string>Minor tickline color</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QFrame" name="MinorTicklineColor_frame">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>60</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>drawMajorXTicklines_checkBox</tabstop>
  <tabstop>drawMajorXTicksLeft_checkBox</tabstop>
  <tabstop>drawMajorXTicksRight_checkBox</tabstop>
  <tabstop>drawMajorXTicksInside_checkBox</tabstop>
  <tabstop>drawMajorXTicksOutside_checkBox</tabstop>
  <tabstop>drawMajorYTicklines_checkBox</tabstop>
  <tabstop>drawMajorYTicksTop_checkBox</tabstop>
  <tabstop>drawMajorYTicksBottom_checkBox</tabstop>
  <tabstop>drawMajorYTicksInside_checkBox</tabstop>
  <tabstop>drawMajorYTicksOutside_checkBox</tabstop>
  <tabstop>drawMinorXTicklines_checkBox</tabstop>
  <tabstop>drawMinorXTicksLeft_checkBox</tabstop>
  <tabstop>drawMinorXTicksRight_checkBox</tabstop>
  <tabstop>drawMinorXTicksInside_checkBox</tabstop>
  <tabstop>drawMinorXTicksOutside_checkBox</tabstop>
  <tabstop>numMinorXTicks_spinBox</tabstop>
  <tabstop>drawMinorYTicklines_checkBox</tabstop>
  <tabstop>drawMinorYTicksTop_checkBox</tabstop>
  <tabstop>drawMinorYTicksBottom_checkBox</tabstop>
  <tabstop>drawMinorYTicksInside_checkBox</tabstop>
  <tabstop>drawMinorYTicksOutside_checkBox</tabstop>
  <tabstop>numMinorYTicks_spinBox</tabstop>
  <tabstop>TickmarkLength_spinBox</tabstop>
  <tabstop>TickmarkWidth_spinBox</tabstop>
  <tabstop>TicklineWidth_spinBox</tabstop>
  <tabstop>MinorTickmarkWidth_spinBox</tabstop>
  <tabstop>MinorTicklineWidth_spinBox</tabstop>
  <tabstop>TickmarkColor_pushButton</tabstop>
  <tabstop>TicklineColor_pushButton</tabstop>
  <tabstop>MinorTicklineColor_pushButton</tabstop>
  <tabstop>OK_pushButton</tabstop>
  <tabstop>Cancel_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
